File: entity/templates/app-java/src/main/java/_Converter.java
package <%= appPackage %>.domain.entity.converter;

import <%= appPackage %>.domain.entity.<%= entityName %>;
import <%= appPackage %>.model.<%= entityName %>;

public class <%= entityName %>Converter {

    public static <%= entityName %> wrap(<%= entityName %>Entity entity) {
        if (entity == null) return null;
        return <%= entityName %>.builder()
                .id(entity.id)
                .build();
    }

    public static <%= entityName %>Entity unwrap(<%= entityName %> model) {
        <%= entityName %>Entity entity = new <%= entityName %>Entity();
        entity.id = model.id();
        return entity;
    }

}

File: entity/templates/app-kotlin/src/main/java/_Converter.kt
package <%= appPackage %>.domain.entity.converter;

import <%= appPackage %>.domain.entity.<%= entityName %>Entity;
import <%= appPackage %>.model.<%= entityName %>;

object <%= entityName %>Converter {

    fun wrap(entity: <%= entityName %>Entity?): <%= entityName %>? {
        if (entity == null) return null
            return <%= entityName %>(entity.id?: 0L)
        }

    fun unwrap(model: <%= entityName %>): <%= entityName %>Entity {
        val entity = <%= entityName %>Entity()
        entity.id = model.id
        return entity
    }

}